synapse/models/inet.py

```diff
@@ -102,6 +103,41 @@ def norm(self, valu, oldval=None):
     def repr(self, valu):
         return valu.encode('utf8').decode('idna')
 
+class Rfc2822Addr(DataType):
+    '''
+    An RFC 2822 compatible email address parser
+
+    Specify strict=1 to strictly enforce complaince.
```

Comment:
Strict does nothing?

Reply:
Whups, meant to remove that when i realized it wasn't useful.  Thanks!